react-toggle-switch-component/dist/styles.css

```diff
@@ -0,0 +1,68 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+#root {
+  position: absolute;
+  margin: auto;
+  top: 50%;
+  left: 50%;
+}
+
+.container {
+  display: flex;
+  align-items: center;
+}
+
+.switch-box {
+  position: relative;
+  display: inline-block;
+  width: 60px;
+  height: 30px;
+}
+
+input {
+  display: none;
+}
+
+.circle {
+  position: absolute;
+  cursor: pointer;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: gray;
+  border-radius: 30px;
+  border: 1px black solid;
+  transition: 0.5s;
+}
+
+.circle::before {
+  position: absolute;
+  content: "";
+  height: 24px;
+  width: 26px;
+  left: 4px;
+  bottom: 2px;
+  background-color: white;
+  -webkit-transition: 0.4s;
+  transition: 0.4s;
+  border-radius: 50%;
+}
+
+input:checked + .circle {
+  background-color: rgb(2, 173, 2);
+}
+
+input:checked + .circle::before {
+  -webkit-transform: translateX(25px);
+  -ms-transform: translateX(25px);
+  transform: translateX(25px);
+}
```

Comment:
If you can explain all of this CSS rule to me, I will accept your PR. For real.
- what is `:checked`
- what is `+`
- what is `::before`
- what is `-webkit-transform`, `-ms-transform`.

Reply:
:checked is a pseudo class that checks if a button is checked. ::before let's you insert content I believe within the html element in relation to the actual content of that element. I know conversely there is also an ::after which does something similar. With both of those then you can use the '+' to select sibling elements. So I wanted to have the circle move once the input box was checked. Webkit and ms transform....honestly I don't know. I've used it before in a lot of the previous assignments as I've seen it come up multiple times.

Reply:
It's a "vendor prefix".
https://developer.mozilla.org/en-US/docs/Glossary/Vendor_Prefix

Reply:
Approved!